constants.xml

```diff
@@ -146,6 +146,7 @@
         <MAX_GOSSIP_MSG_SIZE_IN_BYTES>5000000</MAX_GOSSIP_MSG_SIZE_IN_BYTES>
         <MIN_READ_WATERMARK_IN_BYTES>0</MIN_READ_WATERMARK_IN_BYTES>
         <MAX_READ_WATERMARK_IN_BYTES>20000000</MAX_READ_WATERMARK_IN_BYTES>
+        <CONNECTION_TIMEOUT_IN_SECONDS>1</CONNECTION_TIMEOUT_IN_SECONDS>
```

Comment:
Is 1s timeout too aggressive or it is fine?

Reply:
I think it should be fine. considering we have gossip ROUND_TIME of 1 sec. However, once we do test with large scale (600 shard size) we can re-think of increasing to 2 sec. But i think 1 sec is good enough.